.storybook/storyshots.test.js

```diff
@@ -0,0 +1,18 @@
+import initStoryshots from '@storybook/addon-storyshots';
+
+describe('', () => {
```

Comment:
Can we add a name?

Reply:
The describe is only there to wrap the mock and since the addon already names the tests and I didn't want to add an extra indentation to them..